sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -229,6 +229,44 @@ export interface KeyPhrasesTask {
     modelVersion?: string;
 }
 
+// @public
+export const enum KnownInnerErrorCodeValue {
+    // (undocumented)
```

Comment:
This (undocumented) is part of the generated code that comes from the swagger.

Reply:
This is the new method. For any extensible enums we define it simply as a string and take the allowed values and put it enum just as above. We export both the values